datadog_checks_base/tests/test_openmetrics.py

```diff
@@ -1532,3 +1532,62 @@ def test_text_filter_input(mocked_prometheus_check, mocked_prometheus_scraper_co
 
     filtered = [x for x in check._text_filter_input(lines_in, mocked_prometheus_scraper_config)]
     assert filtered == expected_out
+
+
+@pytest.fixture()
+def mock_filter_get():
+    text_data = None
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'prometheus', 'deprecated.txt')
```

Comment:
nitpick: You can use `get_here` from datadog_checks.dev
```suggestion
    f_name = os.path.join(get_here(), 'fixtures', 'prometheus', 'deprecated.txt')
```

Reply:
thanks for the nitpick